scipy/optimize/_differentialevolution.py

```diff
@@ -1231,10 +1241,11 @@ def solve(self):
                 limits[0, integrality] = DE_result.x[integrality]
                 limits[1, integrality] = DE_result.x[integrality]
 
-            polish_method = 'L-BFGS-B'
+            if 'method' not in self.minimizer_kwargs:
```

Comment:
If we have a default dict containing the method then we don't need this, these two lines can be removed.